Paper 2: Adding source files

anjomshoaa opened the pull request:
## Summary

<!--
TODO: replace `__` with the number of your easychair submission
-->
Submission Number (`SUB_NUMBER`): `2` (*number of your easychair submission*)

<!--
TODO: read and understand the README.md file
https://github.com/AKSW/submission.d2r2.aksw.org/blob/main/README.md
-->

<!--
TODO: make sure finally all boxes are ticked
-->

## Sources
- [x] Created directory `source-management/SUB_NUMBER`
- [x] Added paper sources to `source-management/SUB_NUMBER`
- [x] Verify, the main tex file is called `source-management/SUB_NUMBER/main.tex`
- [x] Verify, the title is properly capitalized according to APA style: https://capitalizemytitle.com/
- [x] Verify, the following correct conference line

```
\conference{Second International Workshop on Linked Data-driven Resilience Research (D2R2'23) co-located with ESWC 2023, May 28th, 2023, Hersonissos, Greece}
```

*Note:* The page limit of 10 pages does not apply anymore. We follow the [submission guidelines by CEUR](https://ceur-ws.org/HOWTOSUBMIT.html).

> We distinguish regular papers (at least 10 "standard" pages) and short papers (5-9 "standard" pages).

## Agreement Form

### First Run
- [x] Created directory `agreement-form/SUB_NUMBER`
- [x] Updated `agreement-form/SUB_NUMBER/metadata.yml`

⏳ Hang on, now the workflow action will run 🎡 to compile all artifacts 📦 and the agreement form 📝 to by signed by you.
(If you are new to github the workflow requires our manual approval.)

When the workflow has finished successfully our robot 🤖 should comment on your pull-request with the links to the artifacts, including the form to be signed.

### Second Run
- [x] Hand-signed the completely filled agreement form ✍️
- [x] One of the following two options:
    - Uploaded a scan of the agreement form to `agreement-form/SUB_NUMBER/agreement.pdf`
    - Send your signed form via email to the editors

> **Mistakes in the editor/author agreements.**<br/>
> Some editors/authors still copy/paste bitmaps of their signatures into the copyright agreements or sign on a touchscreen, instead of signing physically with a pen on paper. Further some authors forget to completely fill out the form.<br/>
> (https://ceur-ws.org/HOWTOSUBMIT.html#TOPERRORS)

- [x] The agreement form is HAND SIGNED with a PEN ON PAPER

## Finally

If all boxes are checked, uncomment the following line.


cc: @white-gecko @seebi @RicardoUsbeck

anjomshoaa committed:
Adding source files.

anjomshoaa committed:
Adding agreement metadata.

anjomshoaa committed:
Update conference name.

anjomshoaa committed:
Adding signed agreement file.

white-gecko commented:
Dear @anjomshoaa thank you for your contribution.
However, I have identified some issues in your camera ready submission:
- in your agreement form's signature field the location is missing
- in your paper the "Acknowledgments" is empty, please add the acknowledgments to your projects resp. funding bodies for your research
- please update the copyrightyear `\copyrightyear{2023}`

anjomshoaa committed:
Location added to the agreement.

anjomshoaa committed:
High quality images added.

anjomshoaa committed:
Delete ZAMG_mapping.png

anjomshoaa committed:
Update main.tex

anjomshoaa commented:
> 

Dear @white-gecko Thanks for the feedback. 
We have addressed the issues and made a new pull request which fails on agreement task : 
"GitHub Actions has encountered an internal error when running your job."
If required, I can send you the agreement per email.
Thanks!

white-gecko commented:
A re-run solved the issue 🤷
